src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -217,6 +240,10 @@ internal virtual HttpRequestMessage GetRequest(Uri uri)
                     }
                     else
                     {
+                        if (stripAuthorization && entry.Key == HttpKnownHeaderNames.Authorization.ToString())
+                        {
```

Comment:
We try to avoid `==` with string comparisons and instead use `string.Equals(a,b, StringComparison.Ordinal)` or the appropriate comparison - this makes it clearer that the author considered culture and case when writing the code.